packages/wonder-blocks-core/src/util/add-style.js.flow

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+import type {StyleType} from "./types";
+
+declare export default function addStyle<
+    T: React.AbstractComponent<any> | string,
+>(
+    Component: T,
+    defaultStyle?: StyleType,
+): React.AbstractComponent<{
+    ...React.ElementConfig<T>,
+    style?: StyleType,
+    ...
+}>;
```

Comment:
I added this file because `flowgen` wasn't able to generate Flow type definitions from the new code that work in webapp.  These types were copied from the original add-style.js file before it was converted to TypeScript.